src/components/buttons/Button.jsx

```diff
@@ -15,27 +15,20 @@ import {
 const Button = props => {
   const { variant, ...buttonProps } = props;
 
-  if (variant === 'primary') {
-    return <PrimaryButton {...buttonProps} />;
+  switch (variant) {
+    case 'primary':
+      return <PrimaryButton {...buttonProps} />;
+    case 'secondary':
+      return <SecondaryButton {...buttonProps} />;
+    case 'gray':
+      return <GrayButton {...buttonProps} />;
+    case 'cancel':
+      return <CancelButton {...buttonProps} />;
+    case 'cancel-secondary':
+      return <CancelSecondaryButton {...buttonProps} />;
+    default:
+      return <DefaultButton {...buttonProps} />;
```

Comment:
What do you think about, making the variant options a enum. And then we can have a hoc that can return `cancelButton[varian]` in that way we could avoid the complete switch. But it is not a blocker we can discuss it in future PR's

Reply:
same, I agree, let's talk to Sergio and we will do it as a refactor for future Hitos

Reply:
how much effort it takes in hours? if it is less than 1 hour please do it and discount time from the next milestone (hito 😬 )